config/releases.exs

```diff
@@ -89,7 +89,8 @@ config :oli,
       ),
     favicons: System.get_env("BRANDING_FAVICONS_DIR", "/favicons")
   ],
-  node_js_pool_size: String.to_integer(System.get_env("NODE_JS_POOL_SIZE", "2"))
+  node_js_pool_size: String.to_integer(System.get_env("NODE_JS_POOL_SIZE", "2")),
+  screen_idle_timeout: String.to_integer(System.get_env("SCREEN_IDLE_TIMEOUT", "30"))
```

Comment:
All of the other timeout variables express their values in milliseconds.  It might make sense to follow that convention and convert `30` to `30000`, or alternatively to rename the variable to include the units: `SCREEN_IDLE_TIMEOUT_IN_SECONDS`

Reply:
I cannot find any other variable that designates type type of timeout, so I think it is fine to introduce a new variable for this.

Reply:
ok, but ideally this should match the session timeout that is set for the whole system or else it's possible that the time could be wrong and the session will still timeout regardless of this helper. 
I assume there *is* a session timeout configured somewhere, perhaps it's just not configured in this way and is part of some other env script? I think we need to keep these 2 values in sync. @eliknebel do you know?

Reply:
The requested variable name changes are done.

Reply:
It sort of depends on whether or not a user is in a persistent session. In either case though I think it uses the POW default. For persistent sessions it looks like this is 30 days https://hexdocs.pm/pow/pow_persistent_session.html#configuration

Reply:
hmm, well I would think the entire reason for even having a feature like this is because students are getting timed out of their sessions while actively working on an adaptive page which is mostly client side and *not* doing a heart beat keep alive. so there must be something somewhere that is configuring this

Reply:
@darrensiegel @eliknebel do you agree? w/ ^^

Reply:
If the adaptive page is making requests and passing the authenticated cookie along then the session should be extended and follow the defaults configured in pow. We use the keepalive heartbeat as a work around for liveviews which get upgraded to a websocket and do not make these requests and therefore were timing out.

I dont think torus has the concept of a session idle timeout anywhere else currently, so it seems reasonable to add a new config for one and perhaps default it to some reasonable value.